.github/CODEOWNERS

```diff
@@ -10,18 +10,19 @@
 # This file
 /.github/CODEOWNERS @edolstra
 
-# Boostraping and core infra
-/pkgs/stdenv                    @edolstra
-/pkgs/build-support/cc-wrapper  @edolstra
-
 # Libraries
 /lib                        @edolstra @nbp
+/lib/systems                @edolstra @nbp @ericson2314
```

Comment:
I overhauled this thing, a lot of stuff there is mainly needed for cross work / less obvious what it is used for outside of that. I'd also "nominate" @dezgeg to be an owner.